build.gradle

```diff
@@ -154,5 +155,21 @@ subprojects {
 
 }
 
+subprojects {
+    // Can't be applied to BOM
+    if (project.name == "kotlinx-serialization-bom") return
```

Comment:
benchmarks could also be excluded here, i think

Reply:
sure, good idea